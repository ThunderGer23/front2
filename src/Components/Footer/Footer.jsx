export const Footer = () => {
  return (
    <>
    <div role='div' className="d-flex align-items-end justify-content-end pe-4">

    </div>
    </>
  )
}
