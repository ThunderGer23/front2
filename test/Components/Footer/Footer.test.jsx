import { render, screen } from "@testing-library/react";
import { Footer } from "../../../src/Components/Footer/Footer";

describe('Pruebas en el componente Footer', () => { 
    test('Esta prueba es para hacer match con el snapshot', () => { 
        const {container} = render(<Footer/>)//Renderiza el componente Footer
        expect(container).toMatchSnapshot();//Toma una foto
     })
 })