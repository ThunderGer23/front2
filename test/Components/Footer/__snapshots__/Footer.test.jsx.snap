// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente Footer Esta prueba es para hacer match con el snapshot 1`] = `
<div>
  <div
    class="d-flex align-items-end justify-content-end pe-4"
    role="div"
  />
</div>
`;
